src/services/telemetry-constants.ts

```diff
@@ -557,6 +557,7 @@ export enum InteractSubtype {
     APPLE = 'apple',
     STATE = 'state',
     COURSE_KEBAB_MENU_CLICKED = 'course-kebab-menu-clicked',
+    CATEGORY_CLICKED = '-clicked'
```

Comment:
CATEGORY_CLICKED = '%-clicked'
and in the code replace the percentile with your value.